jakartaee/10/concurrency/22.0.0.3-beta-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,287 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of Jakarta EE Concurrency.
+
+== Open Liberty 22.0.0.3 Beta - Jakarta EE Concurrency Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/beta/2022-02-01_1901/openliberty-22.0.0.3-beta.zip[Open Liberty 22.0.0.3 Beta]
+
+* Specification Name, Version and download URL:
++
+link:https://jakarta.ee/specifications/Concurrency/3.0[Jakarta EE Concurrency 3.0]
+
+* TCK Version, digital SHA-256 fingerprint and download URL:
++
+Jakarta EE Concurrency 3.0 TCK
+https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-tck/3.0.0/jakarta.enterprise.concurrent-tck-3.0.0.jar[staging]
+| https://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/io/openliberty/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-tck/3.0.0.20220126/jakarta.enterprise.concurrent-tck-3.0.0.20220126.jar[mirrored]
+| https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-tck/3.0.0/jakarta.enterprise.concurrent-tck-3.0.0.pom.sha1[SHA-256]
```

Comment:
I'm not sure that you want to list both the staging and mirrored jar? Which one does the fingerprint apply to? Surely it must be wrong for one of them?

Reply:
It applies to staging.  I included the mirrored copy of the jar as well because Jakarta staging will be deleted in a month or so, and I wanted anyone who looks at the results to still be able to find an artifact somewhere.